docs/transactions/index.md

```diff
@@ -1,4 +1,45 @@
 ---
 sidebar_position: 1
 title: Transactions
----
\ No newline at end of file
+---
+
+Macrometa provides support for user-definable transactions within a region. _Transactions_ allow you to perform multi-document transactions with individual begin, commit, abort commands. They work similar to the BEGIN, COMMIT, and ROLLBACK operations in relational database systems.
+
+## How They Work
+
+Simply put, transactions are a way to aggregate, or group, multiple API calls. The process is as follows:
+
+### 1. Create a Transaction
+
+When you create the transaction, you specify which collections the transaction will write to.
+
+Once you create a transaction, you will be given a transaction identifier.
+
+### 2. Make API Calls
+
+Make API calls that include the transaction identifier to include them in the transaction. All actions will run, but they are not permanent until you commit the transaction.
+
+### 3. Commit or Abort Transaction
+
+- To make all changes made within the transaction permanent, commit the transaction.
+- To roll back the actions, perhaps because one of them failed, abort the transaction.
+
+## Supported Operations
+
+Supported operations include:
+
+- Read and write documents.
+- Get the number of documents in a collection.
```

Comment:
To keep this item consistent with the other sentences in this list, shouldn't we use "in collections" rather than "in a collection"?

Reply:
Where else would they read or write documents to? Are there options besides reading docs in a collection or writing them to a collection?

With the second bullet, "in a collection" is there because it is necessary. Although now I realize you could probably get the number of items in a particular KV Group, too.

Reply:
Documents are only read from and written to collections.

Actually, it is possible to retrieve a list of KV groups in a KV collection within a transaction.

Since in the other items we have "truncate collections" rather than "truncate a collection", and "run queries and query workers" rather than "run a query or a query worker", I was wondering if we shouldn't write it as "get the number of documents in collections" rather than "get the number of documents in a collection". But maybe changing from "in a collection" to "in collections" has any subtle change in the meaning of the sentence that I, as a non-native English speaker, can't get. But it was just a guess.

Reply:
Yes, if I said, "Get the number of documents in collections," that suggests that a user could get the number of documents in all collections, but you can only get the count for one at a time. That's why I changed it to singular.

Reply:
Updated. How's this?